### ISSUE TITLE
[Android] "Failed to send a message {:code-32000}" error occurs when attempting to send audio as result all next sent audios are empty

### ISSUE THREAD

**Issue by VolodLytvynenko** [status-im/status-react#15889]
### Steps to reproduce:
1. Send multiple long text messages to enable scrolling in the chat.
2. Begin recording audio by holding down the 'record' button.
3. Scroll within the chat while the audio is being recorded.
4. Release the 'record' button to stop recording.
5. Send the recorded audio by tapping the  'record' button.

### Actual result:
- The audio recording remains on hold after step 4
- The error message "Failed to send a message {:code-32000}" is displayed after step 5
- Subsequent audio messages sent are empty. (can't be played)

https://github.com/status-im/status-mobile/assets/52490791/c418485c-ebbc-44db-8638-23e7ec07205a

### Expected result:
- The recorded audio should be successfully sent without any error messages
- The audio should be shown as ready to be sent in the composer as soon as 'record' button is released
![image](https://github.com/status-im/status-mobile/assets/52490791/c4f78eca-98c9-43f2-8c5f-7be38d1e6e1a)

- Subsequent audio messages should contain the appropriate audio content.

### Additional info:
The error is not shown for some devices.  As a result, subsequent audio messages will be still sent as empty after steps of this issue
 
### Device:
- Huawei p20 light

### logs:
[Status.log](https://github.com/status-im/status-mobile/files/11464254/Status.log)
https://www.dropbox.com/s/hlt862peoreoa9o/geth.log?dl=0

### Env:
- Nightly [12 May 2023]

**Comment by flexsurfer** [comment]
@briansztamfater  any thoughts?

**Comment by briansztamfater** [comment]
@VladimrLitvinenko does this happen on all Android devices or only on Huawei p20 light?

**Comment by VladimrLitvinenko** [comment]
> @VladimrLitvinenko does this happen on all Android devices or only on Huawei p20 light?

Reproduced on the following devaces:
-  Huawei p20 light
-  iPhone 11 Pro max, IOS 16
- and Android studio as well. Pixel XL, Android 10

**Comment by VolodLytvynenko** [comment]
> @VladimrLitvinenko does this happen on all Android devices or only on Huawei p20 light?

Hi, @briansztamfater yes. @pavloburykh and @qoqobolo were able to successfully reproduce the following points from this issue on their real Android models. 
- The audio recording remains on hold after step 4 https://github.com/status-im/status-mobile/issues/15889#issue-1707678874
- Subsequent audio messages sent are empty. (can't be played)

except for this error reproduced only on Huawei p20 light:
- "Failed to send a message {:code-32000}"
